types/hapi__hapi/index.d.ts

```diff
@@ -4051,7 +4051,8 @@ export namespace Lifecycle {
         (stream.Stream) |
         (object | object[]) |
         symbol |
-        ResponseToolkit;
+        Auth |
+        ResponseObject;
```

Comment:
Why remove `ResponseToolkit` here? Per documentation, it seems that some if its members are returned here:

https://github.com/hapijs/hapi/blob/master/API.md#lifecycle-methods

Where would the toolkit signals and toolkit methods come from here?

Reply:
@saboya not really sure what you're talking about.

If you mean
> a toolkit signal:
h.abandon - abort processing the request.
h.close - abort processing the request and call end() to ensure the response is closed.
h.continue - continue processing the request lifecycle without changing the response.

Then those are all of `symbol` type.

If you mean
> a toolkit method response:
h.response() - wraps a plain response in a response object.
h.redirect() - wraps a plain response with a redirection directive.
h.authenticated() - indicate request authenticated successfully (auth scheme only).
h.unauthenticated() - indicate request failed to authenticate (auth scheme only).

Then those are either of `ResponseObject` or `Auth` type.

The toolkit itself is not the return type, you use the toolkit to return specific `symbol`s or `ResponseObject` or `Auth`.

And as you said
> Per documentation, it seems that some if its members are returned here:

None of it's members types includes the `ResponseToolkit` itself, you do not return the toolkit.

Reply:
You're correct, that was a misinterpration on my part.